mybatis-spring-boot-test-autoconfigure/pom.xml

```diff
@@ -30,18 +30,37 @@
   <name>mybatis-spring-boot-test-autoconfigure</name>
   <dependencies>
     <dependency>
-      <groupId>org.mybatis.spring.boot</groupId>
-      <artifactId>mybatis-spring-boot-starter</artifactId>
+      <groupId>org.springframework.boot</groupId>
```

Comment:
The `spring-boot-starter-test` is `test` scope. I think this is not redundant. What do you think ?